src/components/SRNEntry.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <section class="w-full flex justify-center items-center my-10">
+    <form class="">
+      <div class="form-group">
+        <label class="text-gray-600 font-semibold text-lg">Enter Student ID</label>
```

Comment:
@deepansh96 can comment more again :)

But I would suggest separating styling from the HTML code. The styles should be included in the `<style>` tag so that there is one place to change the CSS, and reuse of same styles is easier

Reply:
No.. It's not about number of styles. The purpose of HTML is content, purpose of JS is additional functionality, and purpose of CSS is styling. We really should follow this separation *as much as possible*. 

Computed properties make sense in the lots of classes case and styles are still separated, but using the `@apply` thing  in tailwind to just add the styles to the `<style>` tag will make separation when these computed properties. I guess we sort of agree:

1. Lots of style: Use tailwind global things or computed properties
2. Few style: Use `@apply` in CSS

But only thing I'm pushing is not to have styles defined the html `<template>` content